new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in feature1 branch
```

Comment:
Its really good that you have decided to add this line of code.

Reply:
Thanks for this comment qadrathejutt!